packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -167,19 +226,20 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
           >
             <div className={classNames.root}>
               {!isModeless && <Overlay isDarkThemed={isDarkOverlay} onClick={isBlocking ? undefined : (onDismiss as any)} />}
-              <FocusTrapZone
-                componentRef={this._focusTrapZone}
-                className={classNames.main}
-                elementToFocusOnDismiss={elementToFocusOnDismiss}
-                isClickableOutsideFocusTrap={isModeless || isClickableOutsideFocusTrap || !isBlocking}
-                ignoreExternalFocusing={ignoreExternalFocusing}
-                forceFocusInsideTrap={isModeless ? !isModeless : forceFocusInsideTrap}
-                firstFocusableSelector={firstFocusableSelector}
-              >
-                <div ref={this._allowScrollOnModal} className={classNames.scrollableContent} data-is-scrollable={true}>
-                  {this.props.children}
-                </div>
-              </FocusTrapZone>
+              {dragOptions ? (
+                <DraggableZone
+                  handleSelector={dragOptions.dragHandleSelector || `.${classNames.main.split(' ')[0]}`}
```

Comment:
Nit, might be worth hoisting this default selector for future maintainers.

Reply:
since this is only done if dragOptions exists, I'm going to leave it here for now

Reply:
Pretty hidden IMHO, but I don't maintain this component.